client/errors.go

```diff
@@ -5,41 +5,48 @@ import (
 	"fmt"
 
 	"github.com/aws/smithy-go"
+
 	"github.com/cloudquery/cq-provider-sdk/provider/schema"
 	"github.com/cloudquery/cq-provider-sdk/provider/schema/diag"
 )
 
 func ErrorClassifier(meta schema.ClientMeta, resourceName string, err error) []diag.Diagnostic {
 	client := meta.(*Client)
+	var ee *diag.ExecutionError
+	if errors.As(err, &ee) {
+		return []diag.Diagnostic{
+			ee,
+		}
+	}
```

Comment:
No need for this its already checked in https://github.com/cloudquery/cq-provider-sdk/blob/be1cd34f3527c627c1c7d814c5f4c90d3979e0aa/provider/provider.go#L247